torchvision/transforms/functional.py

```diff
@@ -3,6 +3,8 @@
 import sys
 import math
 from PIL import Image, ImageOps, ImageEnhance, PILLOW_VERSION
+from scipy.ndimage.interpolation import map_coordinates
+from scipy.ndimage.filters import gaussian_filter
```

Comment:
This makes `torchvision` dependent on `scipy` at import. Up until now we avoided this, so we should discuss if we want to have this now.

Reply:
Firstly, thank you for the quick feedback. 
To answer your question, in all the implementations of elastic transform that I've seen `scipy.ndimage.filters.gaussian_filter` and `scipy.ndimage.interpolation.map_coordinates` were being used unanimously. I understand that this makes `torchvision` dependent on `scipy`, but I have not found any workaround to this yet.

Reply:
You could use create a gaussian filter yourself with the [corresponding equations](https://en.wikipedia.org/wiki/Multivariate_normal_distribution) and replace `map_coordinates` with a variant of [`grid_sample`](https://pytorch.org/docs/stable/nn.functional.html?highlight=grid%20sample#torch.nn.functional.grid_sample). This way no `numpy` or `scipy`.

Reply:
@pmeier Thanks for pointing these out. I'll look into these and get back to you.

Reply:
`gaussian_filter` can be implemented with `Conv` and gaussian weights, here's [the relevant discussion](https://discuss.pytorch.org/t/is-there-anyway-to-do-gaussian-filtering-for-an-image-2d-3d-in-pytorch/12351/7)  

As for map_coordinates, is similar to [`grid_sample`](https://pytorch.org/docs/stable/nn.functional.html?highlight=grid%20sample#torch.nn.functional.grid_sample) as @pmeier said. The interpolation available for `grid_sample` is `bilinear` and `nearest` . Where as `map_coordinates` is interpolated using spline interpolation. If my understanding is correct implementing spline interpolation in `grip_sample` should do the job.